Marlin/Conditionals_LCD.h

```diff
@@ -108,7 +108,28 @@
     #define ULTIPANEL
     #define NEWPANEL
 
+  #elif ENABLED(CR10_STOCKDISPLAY)
+  
+  #define REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
+  #ifndef ST7920_DELAY_1
```

Comment:
as Scott mentioned (and I mentioned earlier) Conditional_LCD.h is loaded prior to Configuration.h so no need to check for pre-defines.

Reply:
I was told I needed them.  :(

Reply:
Now that they're wrapped in conditionals they are defaults — or "recommendations" — and they can be overridden in `Configuration.h` or `Configturation_adv.h`.